### PR TITLE
Extend DeviceInfo & extract ApplicationInfo

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-sdk/lib/integration/http/RelayerApi.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import {http, HttpFunction, PublicRelayerConfig, RelayerRequest} from '@universal-login/commons';
+import {http, HttpFunction, PublicRelayerConfig, RelayerRequest, ApplicationInfo} from '@universal-login/commons';
 import {fetch} from './fetch';
 
 export class RelayerApi {
   private readonly http: HttpFunction;
-  constructor(relayerUrl: string, private applicationName: string) {
+  constructor(relayerUrl: string, private applicationInfo: ApplicationInfo) {
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/integration/http/RelayerApi.ts]: This is terrible place to set `applicationInfo`.
It shoud be set up in sdk configuration. SDK class should pass it as an argument to connect frunction of RelayerAPi.

Comment (xitronix) [reply universal-login-sdk/lib/integration/http/RelayerApi.ts]: We set up these informations in sdk config, and after it we create RelayerApi (sdk class) with applicationInfo, which use in connect and deploy api requests to relayer

Comment (marekkirejczyk) [reply universal-login-sdk/lib/integration/http/RelayerApi.ts]: Still, it ApplicationInfo should not be passed to RelayerAPI.

File: universal-login-sdk/lib/integration/http/RelayerApi.ts [hunk 2]
```diff
@@ -55,14 +55,14 @@ export class RelayerApi {
     return this.http('GET', `/devices/${contractAddress}?signature=${signature}`);
   }
 
-  async deploy(publicKey: string, ensName: string, gasPrice: string, gasToken: string, signature: string) {
+  async deploy(publicKey: string, ensName: string, gasPrice: string, gasToken: string, signature: string, applicationInfo: ApplicationInfo) {
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/integration/http/RelayerApi.ts]: Could be:
```ts
deploy(wallet: DeployedWallet, gasParameters: GasParameters, signature: string, applicationInfo: ApplicationInfo)
```

**Pull request merged**